### ISSUE TITLE
Mini Landing Page Has 2 `<main>` Elements

### ISSUE THREAD

**Issue by bryanpizzillo** [NCIOCPL/cgov-digital-platform#4534]
### Issue description

There are 2 `<main>` elements in the template. The layout should better match https://designsystem.cancer.gov/example-site/inner-pages/inner-page.html. The 2 main elements is wrong because the HTML spec says a page should have only 1. This is an accessibility thing.

Also there should be not content div, nor should there be a div with the legacy `cgdpl` class.

For developer:

- Go to the template and review the issue
- Determine an approach with a plan of what the change should be
- Discuss approach with Bryan prior to making a PR
- With consensus on approach, make the PR
